framework/src/modules/http_api/controllers/transactions.js

```diff
@@ -138,19 +138,24 @@ TransactionsController.postTransaction = function(context, next) {
 	return channel.invoke('chain:postTransaction', [
 		transaction,
 		(err, data) => {
-			if (err) {
-				if (err instanceof ApiError) {
-					context.statusCode = err.code;
-					delete err.code;
-				}
+			let error = null;
+
+			if (data.success) {
+				return next(null, {
+					data: { message: 'Transaction(s) accepted' },
+					meta: { status: true },
+				});
+			}
 
-				return next(err);
+			if (err) {
```

Comment:
It would be cleaner if we do like
`const error = new ApiError(err ? err : data.message, apiCodes.PROCESSING_ERROR);`

Reply:
even better!